iwyu_ast_util.h

```diff
@@ -589,11 +589,11 @@ bool IsDefaultNewOrDelete(const clang::FunctionDecl* decl,
 // Returns true if this decl is part of a friend decl.
 bool IsFriendDecl(const clang::Decl* decl);
 
-// Returns true if a Record-decl looks like a forward-declaration of a
+// Returns true if a named decl looks like a forward-declaration of a
 // class (rather than a definition, a friend declaration, or an 'in
 // place' declaration like 'struct Foo' in 'void MyFunc(struct Foo*);'
 // Always returns false for enums.
```

Comment:
Don't know if we should mention in comment that anonymous `RecordDecl` aren't forward declarations. Leaning more towards not mentioning it because comments don't have to repeat the code and I don't think someone can expect anonymous structure to be a forward declaration.